"""
module : exception handling
creating Custom Exceptions and save as external python files

"""

from stem1400_pythoncore import module_7_exception as err

raise err.CustomError("An error occurred")
